src/Hidden/Hidden.tsx

```diff
@@ -9,18 +10,39 @@ export type HiddenProps = {
   children: React.ReactNode
 }
 
+function isArray(value: Array<string> | string): value is Array<string> {
+  return Array.isArray(value)
+}
+
 export const Hidden = ({on: hiddenViewports, children}: HiddenProps) => {
```

Comment:
This is unrelated to this PR, but from a naming scheme `on` for breakpoints feels like an awkward phrase to me. Typically `on` is used for actions/events (like event handlers) and `when` or `only` is used for conditions (like breakpoints). I'm curious if there would be interest in revisiting this and using terms like:

```tsx
<Hidden when="narrow" />
<Hidden above="narrow" />
<Hidden below="wide" />
<Hidden breakpoint="wide" />
<Hidden breakpoints={['narrow', 'regular']} />
```

Reply:
Sure, I am happy to re-think and discuss about it. Just to clarify that it seems like we have [breakpoints](https://primer.style/primitives/spacing#breakpoints) and [viewport ranges](https://primer.style/primitives/spacing#viewport-ranges) and we use viewport ranges for the `on` prop.

Considering that, using `when` makes sense to me. Tagging @mperrotti and @colebemis to get their opinion too 🙌🏻